api-docs/leads.md

```diff
@@ -0,0 +1,332 @@
+# Leads
+
+The leads document direct encounters with leads.
+
+Leads are potiental clients.
+
+Here is the list of the valid lead types:
+
+- `leadCreated`,
+- `leadUpdated`,
+- `leadDeleted`
+
+## Get a single Lead
+
+### Request
+
+```http
+GET /api/leads/:id
+```
+
+### Response
+
+```json
+{
+  "id": 1,
+  "dateOfSignup": "2019-09-17",
+  "leadStatus": "pending",
+  "firstName": "Joel",
+  "lastName": "Denning",
+  "fullName": "Joel Denning",
+  "phone": "555-555-5555",
```

Comment:
Let's remove the dashes. This not only matches the clients API, but also makes it easier for code to know how to format the phone number. If there are dashes, there might also be parentheses and space characters. Better to just get the raw numbers (as a string)